apps/examples/kernel_stability_test/Kconfig

```diff
@@ -0,0 +1,15 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_KERNEL_STABILITY_TEST
+	bool "Kernel Stability Test Sample"
```

Comment:
Could you explain more about this app?
In addition, this sample app doesn't cover the whole kernel stability, but only a part of it.
So, I just recommend you to change the name of this test app to 'HEAVY_SIGNAL_MESSAGE_TEST'.
The description might be 
"Aging test in case of heavy signal/message scenarios."

Reply:
Thanks. I will modify.